irods/__init__.py

```diff
@@ -92,7 +92,7 @@ def client_logging(flag=True, handler=None):
 
 PAM_AUTH_PLUGIN = "PAM"
 PAM_AUTH_SCHEME = PAM_AUTH_PLUGIN.lower()
-PAM_AUTH_SCHEMES = (PAM_AUTH_SCHEME, "pam_password")
+PAM_AUTH_SCHEMES = (PAM_AUTH_SCHEME, "pam_password", "pam_interactive")
```

Comment:
This caused the "viewed" checkbox to be reset for me. Did something actually change in the PR?